tests/main.nf.test

```diff
@@ -120,6 +120,22 @@ nextflow_pipeline {
 
                 skip_allele_calling = true
 
+                QCReport {
+                    fallthrough {
```

Comment:
Thanks so much for adding this. This looks great 😄 

I am wondering if you could add another test case where the average coverage quality check fails?

And maybe another test case where `fixed_genome_size` is modified to show that the number is being passed through properly?

Reply:
ah that would be an excellent idea! I will add one now!!

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/102/commits/aa889d85917670951646647930dcf7d1d2330ba9

Reply:
Thanks so much.